web/packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -181,6 +183,18 @@ export class DocumentsService {
     };
   }
 
+  createConnectMyComputerSetupDocument(
+    opts: CreateConnectMyComputerSetupDocumentOpts
```

Comment:
Some time ago I wanted to remove all those `create*Document` from `DocumentsService` and make them top-level functions (unfortunately I didn't have time to do that). IMO this makes sense, since they only return objects based on the provided parameters. In light of this idea, it would be better not to create a relationship between `DocumentsService` and `createConnectMyComputerSetupDocument`.